junebug/command_line.py

```diff
@@ -27,6 +26,42 @@ def parse_arguments(args):
     parser.add_argument(
         '--log-file', '-l', dest='logfile', default=None, type=str,
         help='The file to log to. Defaults to not logging to a file')
+    parser.add_argument(
+        '--redis-host', '-redish', dest='redis_host', default='localhost',
+        type=str,
+        help='The hostname of the redis instance. Defaults to "localhost"')
+    parser.add_argument(
+        '--redis-port', '-redisp', dest='redis_port', default=6379,
+        type=int,
+        help='The port of the redis instance. Defaults to "6379"')
+    parser.add_argument(
+        '--redis-db', '-redisdb', dest='redis_db', default=0,
+        type=int,
+        help='The database to use for the redis instance. Defaults to "0"')
+    parser.add_argument(
+        '--redis-password', '-redispass', dest='redis_pass', default=None,
+        type=str,
+        help='The password to use for the redis instance. Defaults to "None"')
+    parser.add_argument(
+        '--amqp-host', '-amqph', dest='amqp_host', default='127.0.0.1',
+        type=str,
+        help='The hostname of the amqp endpoint. Defaults to "127.0.0.1"')
+    parser.add_argument(
+        '--amqp-vhost', '-amqpvh', dest='amqp_vhost', default='/',
+        type=str,
+        help='The amqp vhost. Defaults to "/"')
+    parser.add_argument(
+        '--amqp-port', '-amqpp', dest='amqp_port', default=5672,
+        type=int,
+        help='The port of the amqp endpoint. Defaults to "5672"')
+    parser.add_argument(
+        '--amqp-user', '-amqpu', dest='amqp_user', default='guest',
+        type=str,
+        help='The username to use for the amqp auth. Defaults to "guest"')
+    parser.add_argument(
+        '--amqp-password', '-amqppass', dest='amqp_pass', default='guest',
+        type=str,
+        help='The password to use for the amqp auth. Defaults to "guest"')
```

Comment:
I was wondering about this. It started with command line args, so I just continued it. Maybe we could do both? I just thought it might be strange to create and specify a config file just to change the port that junebug is running. Maybe we could have both (in that you have a `--config-file=..`, or you can specify via the command line)